code/game/world.dm

```diff
@@ -271,7 +271,7 @@ GLOBAL_VAR(restart_counter)
 			TgsEndProcess()
 
 	log_world("World rebooted at [time_stamp()]")
-	
```

Comment:
In case anyone asks, my first Mentorhelp PR did this, this PR is just undoing what the first one did.